.env.schema

```
@@ -0,0 +1,30 @@
+# none: no authentication
+# azure: use the azure ad authentication see https://dcc-bs.github.io/documentation/nuxt-layers/auth.html#azure-auth-implementation-azure-auth
+# @type=enum(none, azure)
+AUTH_MODE="none"
+
+# True to use the Feedback control
+# @type=boolean
+USE_FEEDBACK=true
+
+# Gets calulcated from the APP_MODE
+# @type=boolean
+IS_RUNTIME=not(eq($APP_MODE, build))
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)